README.md

```diff
@@ -1 +1,46 @@
-# starter-kit-storage-and-interfaces
\ No newline at end of file
+# starter-kit-storage-and-interfaces
+
+To boot strap all of this simply run:
+
+```cmd
+docker compose up -d
+```
+
+For demonstration purpose with a example dataset use the following command:
+
+```cmd
+ocker compose --profile demo up -d
+```
+
+The files imported by the data loading script comes from here: `https://github.com/ga4gh/htsget-refserver/tree/main/data/gcp/gatk-test-data/wgs_bam`
+
+## Download unencrypted files directly
+
+### Get token for downloading data
+
+There are 3 tokens available, the first has access to a dataset at this site, the second one is empty and the third doesn't have access to this site.
+
+```cmd
+token=$(curl -s -k https://localhost:8080/tokens | jq -r .[0])
```

Comment:
```suggestion
token=$(curl -s -k https://localhost:8080/tokens | jq -r '.[0]')
```

Quoting the `jq` expressions prevents errors in `bash` sessions that use `failglob`, or in `zsh` (a similar option is set by default), and also avoids accidentally globbing a file called `.0` if there is one.